source/NVDAObjects/window/winword.py

```diff
@@ -152,6 +152,14 @@ def _displayStringLabels(self) -> dict[Self, str]:
 wdAlignParagraphCenter = 1
 wdAlignParagraphRight = 2
 wdAlignParagraphJustify = 3
+
+
+# From WdOutlineLevel enumeration
+# See https://learn.microsoft.com/en-us/office/vba/api/word.wdoutlinelevel
+class WdOutlineLevel(IntEnum):
+	BODY_TEXT = 10
```

Comment:
Documentation as part of enum, please
```suggestion
class WdOutlineLevel(IntEnum):
	"""Outline level of a paragraph.

	.. seealso:: ```WdOutlineLevel`` enumeration <https://learn.microsoft.com/en-us/office/vba/api/word.wdoutlinelevel>`_
	"""
	
	BODY_TEXT = 10
```